rook/deploy-rook.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+helm repo add rook-release https://charts.rook.io/release
+
+namespace="rook-ceph"
+release_name="rook-ceph"
+chart="rook-release/rook-ceph"
+chart_version="v1.5.3"
+
+# Install rook operator
+kubectl create namespace $namespace
```

Comment:
I actually think we should have this as a file and use `apply -f` since we cannot run this multiple times atm.

Reply:
:+1:

Reply:
Sure, changed it to `kubectl create | apply` though since it's a plain namespace object :slightly_smiling_face: